addon_toolkit/imp.py

```diff
@@ -87,7 +87,9 @@ async def invoke_operation(
         if not inspect.iscoroutinefunction(_operation_method):
             _operation_method = sync_to_async(_operation_method)
         _result = await _operation_method(**_kwargs)
-        assert isinstance(_result, operation.result_dataclass)
+        assert isinstance(
+            _result, operation.result_dataclass
+        ), f"expected {operation.result_dataclass.__name__} type to bre returned from method {_operation_method.__name__}, got {_result.__class__.__name__}"
```

Comment:
This was mentioned in the Google Drive review, but 
```suggestion
        ), f"expected {operation.result_dataclass.__name__} type to be returned from method {_operation_method.__name__}, got {_result.__class__.__name__}"
```